### ISSUE TITLE
Localization textLabels values as React components

### ISSUE THREAD

**Issue by yaminyaylo** [gregnb/mui-datatables#1401]
Currently, I have an issue in console

`index.js:1 Warning: validateDOMNesting(...): div cannot appear as a descendant of p. in div (created by ForwardRef(CircularProgress))`

that's because I'm using it =>
```
        textLabels: {
            body: {
                noMatch: isLoading ? (
                    <CircularProgress color='secondary' component='span' />
                ) : (
                    texts.noPricingRecordsFound
                )
            }
        },

```

for a long long time, but only today (after upgrade to v3.1.4) it starts complaining maybe because of validation was implemented in that version.

<img width="1258" alt="Screenshot 2020-07-10 at 17 01 07" src="https://user-images.githubusercontent.com/1785530/87163799-952d9500-c2d0-11ea-8020-950b55682013.png">
<img width="1261" alt="Portal 2020-07-10 17-01-32(1)" src="https://user-images.githubusercontent.com/1785530/87163882-ad9daf80-c2d0-11ea-8188-74c5add958e2.png">

Can we suppose to support passing React elements and not only text strings inside the Table textLabels localization?

**Comment by patorjk** [comment]
Thank you for the report. I've put a fix for this in v3.2.0, which should come out next week and is available in beta here:

    npm install mui-datatables@v3.2.0-beta.0

**Comment by patorjk** [comment]
Fixed in version 3.2.0

**Comment by yaminyaylo** [comment]
@patorjk thanks a lot!
